src/appleseed/foundation/image/genericprogressiveimagefilereader.cpp

```diff
@@ -177,7 +184,9 @@ void GenericProgressiveImageFileReader::close()
     // but OpenImageIO 1.5.20 (the version included in appleseed-deps at the time
     // of writing) is too old to have this method. Since on Windows we link to
     // OpenImageIO statically, this should be safe anyway.
+#if OIIO_VERSION < 20000
```

Comment:
Not strictly part of your PR, but... Is the comment above still relevant? If it is and we keep the comment, then it should be inside the `#if ... #endif` block.

Reply:
Not relevant anymore, and OIIO::ImageInput::destroy() can be called.